### PR TITLE
Transformation fixes

### PR DESCRIPTION
**Pull request by jamezp**
* https://issues.redhat.com/browse/WFLY-16497
* https://issues.redhat.com/browse/WFLY-16498

Also a couple other clean ups.

### CODE REVIEW

File: ee-9/feature-pack/pom.xml [hunk 1]
```diff
@@ -538,18 +541,21 @@
...
-                                <excluded>org.apache.cxf:cxf-rt-databinding-jaxb\z</excluded>
-                                <excluded>org.apache.cxf:cxf-rt-frontend-simple\z</excluded>
-                                <excluded>org.apache.cxf:cxf-rt-transports-jms\z</excluded>
-                                <excluded>org.apache.cxf:cxf-rt-transports-http\z</excluded>
-                                <excluded>org.apache.cxf:cxf-rt-ws-rm\z</excluded>
-                                <excluded>org.apache.cxf:cxf-tools-common\z</excluded>
-                                <excluded>org.apache.cxf:cxf-tools-java2ws\z</excluded>
+                                <exclude>org.hibernate.orm:hibernate-core\z</exclude>
+                                <exclude>org.hibernate.orm:hibernate-envers\z</exclude>
+                                <exclude>org.hibernate.search:hibernate-search-mapper-orm-orm6\z</exclude>
```
Comment (scottmarlow) [inline ee-9/feature-pack/pom.xml:558]: :+1: for the Hibernate ORM excludes!

File: ee-9/feature-pack/pom.xml [hunk 2]
```diff
@@ -817,6 +823,10 @@
                     <groupId>${project.groupId}</groupId>
                     <artifactId>jipijapa-eclipselink</artifactId>
                 </exclusion>
+                <exclusion>
+                    <groupId>${ee.maven.groupId}</groupId>
+                    <artifactId>jipijapa-hibernate5-3</artifactId>
```
Comment (scottmarlow) [inline ee-9/feature-pack/pom.xml:828]: :+1: for excluding the jipijapa-hibernate5-3, soon that will be removed but not yet.

File: ee-9/feature-pack/pom.xml [hunk 3]
```diff
@@ -959,6 +977,14 @@
                     <groupId>org.jboss</groupId>
                     <artifactId>jboss-ejb-client</artifactId>
                 </exclusion>
+                <exclusion>
+                    <groupId>org.hibernate</groupId>
+                    <artifactId>hibernate-core</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>org.hibernate</groupId>
```
Comment (scottmarlow) [inline ee-9/feature-pack/pom.xml:985]: +1

**Review by scottmarlow — APPROVED** [review]

File: ee-feature-pack/common/src/main/resources/modules/system/layers/base/org/jboss/narayana/compensations/main/module.xml [hunk 4]
```diff
@@ -29,7 +29,7 @@
     </properties>
 
     <resources>
-        <artifact name="${org.jboss.narayana.compensations:compensations}"/>
+        <artifact name="\${org.jboss.narayana.compensations:compensations@module.jakarta.suffix@}"/>
```
Comment (mmusgrov) [inline ee-feature-pack/common/src/main/resources/modules/system/layers/base/org/jboss/narayana/compensations/main/module.xml:32]: Thanks, and similarly for the other XTS and RTS ones.

**Review by mmusgrov — APPROVED** [review]
Thanks for the heads up.

**Pull request merged**